sessions/GraphQL-100/102/client/README.md

```diff
@@ -4,6 +4,10 @@ The following tasks give you exposure to working with graphql on the client side
 
 Tasks:
 
-1. Fetch all products from the graphql server (components/ProductList.js:19)
-2. Fetch a product from the graphql server (components/ProductPage:19)
-3. Fetch all orders from the grapqhl server (component/OrdersPage:10)
+1. Fetch all products from the graphql server (components/ProductList.js)
+2. Fetch a product from the graphql server (components/ProductPage.js)
+3. Create a new order (components/ProductPage.js)
+4. Fetch all orders from the grapqhl server (component/OrdersPage.js)
+
+
```

Comment:
Should this have instructions for running the `npm server` and the build and run of the client?